float64 com_height
float64[] default_joint_state